src/@ionic-native/plugins/google-maps/index.ts

```diff
@@ -687,21 +663,53 @@ export class BaseClass {
    *
    * @return {Observable<any>}
    */
-  @CordovaInstance({
```

Comment:
I see that you've removed some decorators. I suggest you replace them with `CordovaCheck` and `InstanceCheck` decorators. These two decorators check if the plugin/objectInstance exists before executing the functionality. This helps eliminate runtime errors when we're developing outside of a Cordova platform.